index.html

```diff
@@ -27,15 +27,16 @@
     <title>Francisco's Portfolio</title>
 </head>
 <body>
-        <header class="nav-bar">
+        <header id="nav" class="nav-bar">
             <h3>Francisco Ponce</h3>
+            <img id="close" src="./assets/closing-btn.svg" alt="Close menu" class="closing-btn">
             <ul class="desktop-only">
                 <li><a class="link" href="#portfolio">Portfolio</a></li>
                 <li><a class="link" href="#about-me">About</a></li>
                 <li><a class="link" href="#contact">Contact</a></li>
                 <li><a class="link" href="#contact"><i class="far fa-envelope fa-lg"></i></a></li>
```

Comment:
- [ ] You are currently missing the close button for the hamburger popup. If you look at the Figma design you'll notice that this is supposed to be preset. please implement this design.
- [ ] Implement the functionality for closing the hamburger when you click on the close icon.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/150874005-857c1e68-ac86-4cc4-9f98-436953d4de66.png)|![image](https://user-images.githubusercontent.com/18730106/150873818-18a6fbd7-e08a-4cf5-8788-7c4058366ad1.png)